From e4ea25a968b52353a354920e4e95cc30ae3130dc Mon Sep 17 00:00:00 2001
From: Rafael Palomar <rafael.palomar@rr-research.no>
Date: Thu, 12 Sep 2019 06:41:55 +0200
Subject: [PATCH] ENH: Add check for finding CTKAppLauncherLib

Base/QTCore/CMakeLists.txt will look for
CTKAppLauncherLib (required). This would produce CMake configuration
errors with Slicer_SUPERBUILD=OFF and Slicer_USE_CTKAPPLAUNCHER=OFF.
---
 Base/QTCore/CMakeLists.txt | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/Base/QTCore/CMakeLists.txt b/Base/QTCore/CMakeLists.txt
index 6c55787ee..cd62ab757 100644
--- a/Base/QTCore/CMakeLists.txt
+++ b/Base/QTCore/CMakeLists.txt
@@ -42,7 +42,9 @@ endif()
 #
 # CTKAppLauncherLib
 #
-find_package(CTKAppLauncherLib REQUIRED)
+if(Slicer_USE_CTKAPPLAUNCHER)
+  find_package(CTKAppLauncherLib REQUIRED)
+endif()
 
 #
 # See CMake/SlicerMacroBuildBaseQtLibrary.cmake for details
-- 
2.21.0

